doc/rados/operations/pools.rst

```diff
@@ -316,7 +316,11 @@ You may set values for the following keys:
 
 :Description: Sets the minimum number of replicas required for I/O.
               See `Set the Number of Object Replicas`_ for further details.
-              Replicated pools only.
+              In the case of Erasure Coded pools this should be set to a value
+              greater than 'k' since if we allow IO at the value 'k' there is no
```

Comment:
how about

> greater or equal to 'k'

?

Reply:
unless the data is unimportant/you have strangely reliable hardware, min_size=k (or 1 for replication) should only be used in emergencies. given how many folks have had issues with min_size=1, I'd hesitate to recommend it in the docs

Reply:
@tchaikov people setting min_size to 'k' is how this doc review came about. The recommended value is actually k+2 per https://docs.ceph.com/docs/master/rados/operations/erasure-code/#erasure-coded-pool-recovery

Reply:
thanks gentlemen! just realized i was proposing a recipe of data lost !

Reply:
@tchaikov no problem!